### PR TITLE
Implement cursor_shape

### PR DESCRIPTION
**Pull request by Nefsen402**

### CODE REVIEW

File: protocol/meson.build [hunk 1]
```diff
@@ -17,6 +17,8 @@ wayland_scanner_client = generator(
 
 client_protocols = [
 	wl_protocol_dir / 'stable/xdg-shell/xdg-shell.xml',
+	wl_protocol_dir / 'staging/cursor-shape/cursor-shape-v1.xml',
+	wl_protocol_dir / 'unstable/tablet/tablet-unstable-v2.xml',
```
Comment (emersion) [inline protocol/meson.build:21]: Nit: we don't need this protocol

Comment (Nefsen402) [reply protocol/meson.build:21]: A linking error occurs without it.

Comment (emersion) [reply protocol/meson.build:21]: Ah, right. Probably an access to the `zwp_tablet_tool_v2_interface` global variable.

**Review by emersion — APPROVED** [review]
Thanks!

**Pull request merged**